cidc_api/resources/downloadable_files.py

```diff
@@ -86,12 +86,10 @@ def get_related_files(downloadable_file: DownloadableFiles):
     return {"_items": downloadable_file.get_related_files()}
 
 
-@downloadable_files_bp.route("/filelist", methods=["GET"])
+@downloadable_files_bp.route("/filelist", methods=["POST"])
```

Comment:
just curious why not keep both and not make it a breaking change?

Reply:
My thinking is, the current code is already broken for large file batches, so a breaking change doesn't hurt *that much* more 🤷

It's true that keeping both wouldn't really hurt either, aside from potentially causing confusion.